front/src/config/i18n/en.json

```diff
@@ -1489,6 +1489,36 @@
       "acceleration-z": "Acceleration Z",
       "bed-activity": "Bed Activity"
     },
+    "television": {
+      "shortCategoryName": "Television",
+      "binary": "Power",
+      "source": "Source",
+      "channel": "Channel",
+      "guide": "Guide",
+      "menu": "Menu",
+      "volume-up": "Volume up",
+      "volume-mute": "Mute",
+      "volume-down": "Volume down",
+      "volume": "Volume",
+      "channel-up": "Channel up",
+      "channel-previous": "Previous channel",
+      "channel-down": "Channel down",
+      "tools": "Tools",
```

Comment:
is this common to all TVs?

Reply:
ah I see below - perhaps this should be `Settings`?

Reply:
On my Samsung remote, I see "TOOLS", but settings is acceptable.
I'll wait when this is ready for review.